core/models.py

```diff
@@ -27,6 +29,14 @@ class MostKnown(models.Model):
     #Timestamp for each entry for historical graphing
     date_time = models.DateField('Timestamp')
 
+    class Meta:
+        verbose_name = "Most Know Colleague"
+        verbose_name_plural ="Most known Colleagues"
+
```

Comment:
Minor edit - Spelling for 'Known' & for the plural consistency for capitals 'know' -> 'Known'